### PR TITLE
Soft delete

### PR DESCRIPTION
**Pull request by dianaboiangiu**

### CODE REVIEW

File: insitu/models.py [hunk 1]
```diff
@@ -103,7 +161,7 @@ def __str__(self):
         return self.name
 
 
-class ProductRequirement(models.Model):
+class ProductRequirement(SoftDeleteModelMixin):
```
Comment (andreeadima) [inline insitu/models.py]: Change the naming - mix-in means extra behaviour on something existent, so a class can never inherit from mix-in. Maybe SoftDeleteModel?

File: insitu/static/css/style.css [hunk 2]
```diff
@@ -26,3 +26,7 @@
 .paginate_button {
     padding: 0 !important;
 }
+
+.delete-btn{
+    margin-left: 1%
+}
```
Comment (andreeadima) [inline insitu/static/css/style.css]: New line

File: insitu/templates/data_group/delete.html [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+{% block title %}
+  Delete {{ data_group.name }}
+{% endblock %}
+
+{% block content %}
+
+  <div class="row">
+    <h1 class="col-sm-12">
+      <span>Delete {{ data_group.name }}</span>
```
Comment (andreeadima) [inline insitu/templates/data_group/delete.html]: Should we style this as we did with the other headings? The name in <small>

**Comment by dianaboiangiu** [comment]
Requested changes made.

**Pull request merged**